resize-observer/devicepixel2-ref.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<body>
+<script>
+const patternSize = 4;
+
+function createPatternDataURL() {
+  const ctx = document.createElement('canvas').getContext('2d');
+  ctx.canvas.width = patternSize;
+  ctx.canvas.height = patternSize;
+
+  const b = [0, 0, 0, 255];
+  const t = [0, 0, 0, 0];
+  const r = [255, 0, 0, 255];
+  const g = [0, 255, 0, 255];
+
+  const imageData = new ImageData(4, 4);
```

Comment:
`s/4/patternSize/`

(for consistency with the testcase, where I think you fixed this previously)